Source/ACE.Server/Command/Handlers/DeveloperDatabaseCommands.cs

```diff
@@ -139,5 +145,119 @@ public static List<string> OutputShortcutSQLCommand(string playerName, uint char
 
             return strings;
         }
+
+        [CommandHandler("import", AccessLevel.Developer, CommandHandlerFlag.None, 1, "Imports a weenie from the Content folder", "<wcid>")]
+        public static void HandleImport(Session session, params string[] parameters)
```

Comment:
Instead of DeveloperDatabaseCommands, i think this stuff should go into ContentCreationCommands

Reply:
the structure of Command/Processors is already getting a little unruly

Handlers/ should ideally be organized by AccessLevel. If we want to further divide up a particular access level into multiple files, then it should still be prefixed with the base AccessLevel, ie. AdminCommands.cs, AdminShardCommands.cs, etc.

Currently there is no 'ContentCreator' access level (although this could be a possible idea in the future). If I am looking for all of the Developer commands, and they are split between Developer*Commands.cs and a bunch of other unrelated files, this could also be confusing

The Command folder and UI will be getting a refactoring pass very soon in general

Reply:
config.js is for things that are required to start up the server

content_folder is not required to start up the server. it is also something that won't be used for normal server operation, only for content editing

PropertyManager also gives the user the ability to change content_folder on the fly

Reply:
Access level really should be a flags enum, and grouping of commands should be based on function, not access level.

In the future, commands should support multiple access levels.

Reply:
should this go into DeveloperContentCreatorCommands.cs?

Reply:
added DeveloperContentCommands.cs